idaes/core/base/tests/test_control_volume_0d.py

```diff
@@ -2280,6 +2280,24 @@ def test_add_total_energy_balances():
         m.fs.cv.add_total_energy_balances()
 
 
+@pytest.mark.unit
+def test_add_isothermal_energy_balances():
+    m = ConcreteModel()
+    m.fs = Flowsheet(dynamic=False)
+    m.fs.pp = PhysicalParameterTestBlock()
+    m.fs.rp = ReactionParameterTestBlock(property_package=m.fs.pp)
+    m.fs.pp.del_component(m.fs.pp.phase_equilibrium_idx)
```

Comment:
Why is this necessary?
```suggestion
    m.fs.pp.del_component(m.fs.pp.phase_equilibrium_idx)
```

Reply:
Cleaned up